app/src/ui/app.tsx

```diff
@@ -1524,6 +1524,7 @@ export class App extends React.Component<IAppProps, IAppState> {
             key="preferences"
             initialSelectedTab={popup.initialSelectedTab}
             dispatcher={this.props.dispatcher}
+            accounts={this.state.accounts}
             dotComAccount={this.getDotComAccount()}
```

Comment:
Just curious: any reason for not getting rid of this too?

Reply:
They'll be gone in the next PR I believe. They're used by the GitConfig tab so I'm gonna tackle that separately from the Accounts tab is all.